src/controllers/middleware.js

```diff
@@ -0,0 +1,16 @@
+const { parse } = require('cookie');
+const { verify } = require('jsonwebtoken');
+require('env2')('./config.env');
+
+exports.authCheck = (request, response, next) => {
```

Comment:
Fix indentation, and while you're here you could try adding a few comments to describe the function and the steps it takes.